apps/swirl-docs/src/components/Documentation/Parameter.tsx

```diff
@@ -42,11 +63,61 @@ export function Parameter({
               />
             ),
           }}
-          className="text-sm text-text-default"
+          className="text-sm text-text-default mt-2"
         >
           {description}
         </ReactMarkdown>
       )}
+      {enumValues?.length && (
+        <div className="mt-space-8">
```

Comment:
**nitpick (non-blocking):** i implemented framer motion as a animation lib already (see sidebar navigation / category navigation) for reference. For the UX i think it would be a cool addition to also see it here. should not be that big of an effort :)

Reply:
Done.